lib/matplotlib/pyplot.py

```diff
@@ -1012,19 +1012,20 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
             - 'col': each subplot column will share an x- or y-axis.
 
         When subplots have a shared x-axis along a column, only the x tick
-        labels of the bottom subplot are visible.  Similarly, when subplots
+        labels of the bottom subplot are created. Similarly, when subplots
         have a shared y-axis along a row, only the y tick labels of the first
-        column subplot are visible.
+        column subplot are created. To later turn other subplots' ticklabels
+        on, use :meth:`~matplotlib.axes.Axes.tick_params`.
 
     squeeze : bool, optional, default: True
-        - If True, extra dimensions are squeezed out from the returned Axes
-          object:
+        - If True, extra dimensions are squeezed out from the returned
+          array of Axes object:
```

Comment:
plural

Reply:
I really shouldn't have touched this squeeze parameter at all. :laughing: 

Anyways, as I understand the sentence it is "Extra dimensions are squeezed out from the returned
 object." (singular)  
Now which object is that more specifically? It is an (array of Axes)-object.

If that should not be the case, could you please suggest the propper wording, because not being a native speaker, I wouldn't know how to write this differently.

Reply:
My comment was unclear.  I meant "array of Axes object" -> "array of Axes objects".

Reply:
In that case my above comment would still apply. `subplots` returns one single array. That array is one single object. 

But I now understand the issue. So I guess both ways are correct, depending on what you want to say.

This can be an (array of Axes)-object.
Or it can be an array of (Axes-objects).

Reply:
What about leaving the object out completely?

Reply:
I’d say “extra dimensions from the two-d array of axes are squeezed out.”